lib/sphinx/integration/helper.rb

```diff
@@ -72,7 +77,17 @@ def index
           ::Sphinx::Integration::ReplayerJob.enqueue(index.core_name)
         end
       end
+      begin
+        Rails.application.config.sphinx_integration.fetch(:send_index_notification).try(:call, SUCCESS)
+      rescue StandardError => err
```

Comment:
Lint/HandleExceptions: Do not suppress exceptions.<br>Style/RescueStandardError: Avoid rescuing without specifying an error class.